python_scripts/ensemble_gradient_boosting.py

```diff
@@ -130,7 +130,8 @@ def generate_data(n_samples=50):
 # select this sample in `data_train`.
 
 # %%
-data_max = data_train.iloc[-2, 0]
+data_max = data_train.iloc[[-2]]
+data_max.reset_index(drop=True, inplace=True)
```

Comment:
Can we avoid resetting the index here? Do we use it somewhere?

Reply:
I use it to easily access the first element of data_max in [line 197](https://github.com/INRIA/scikit-learn-mooc/blob/25a73b6dfa188d215bad24c6dd2442198091aefc/python_scripts/ensemble_gradient_boosting.py#L197)

Reply:
we could always select by position with `.iloc[0]`?

Reply:
Done, thanks!